examples/telegraf/Dockerfile.spdk

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) Arm Limited and Contributors
+# Copyright (c) Intel Corporation
+FROM fedora:33
```

Comment:
Any reason to use a Fedora version this old? Fedora 33 was EOL'd on November 30, 2021, as you can see on the [wiki](https://en.wikipedia.org/wiki/Fedora_Linux_release_history).

Reply:
no reason at all, just used the existing dockerfile from https://github.com/spdk/spdk-csi/blob/master/deploy/spdk/Dockerfile

with new release of SPDK `v22.05` I will re-test and update both SPDK version and Fedora version

Reply:
FYI: I opened an IPDK issue for this as well, as it seems to use Fedora 33 liberally everywhere and should also be updated: https://github.com/ipdk-io/ipdk/issues/134